static/usage/v7/input/theming/css-properties/vue.md

```diff
@@ -1,6 +1,13 @@
 ```html
 <template>
-  <ion-input aria-label="Custom input" placeholder="Custom input" class="custom"></ion-input>
+  <ion-input
+    aria-label="Custom input" 
+    placeholder="Custom input" 
+    class="custom"
+    helper-text="Helper text"
```

Comment:
The helper text doesn't display when opening the Vue Stackblitz; Ionic bug? 🤔

Reply:
It's working on my end 
![image](https://user-images.githubusercontent.com/2721089/217340354-7da32bb3-04fc-4bc3-8239-648546a2c6c7.png)

Reply:
Looks like it's only visible in dark mode for some reason? I noticed it's rendering, but not visible on my end because the text is white:
![white text](https://i.gyazo.com/35a2e6014e43ff709ce8d4d4f4d340bf.png)

Reply:
Looks like the styles were being scoped so they did not apply. Fixed in https://github.com/ionic-team/ionic-docs/pull/2759/commits/0719ef436234a211a256793af9d4c9012bd4b0f2. I also improved the contrast in dark mode.